recipes/rstudio/meta.yaml

```diff
@@ -0,0 +1,177 @@
+    script: build-desktop.sh
+    build:
+      skip: true  # [win]
+      rpaths:
+        - lib/R/lib/
+        - lib/rstudio/lib
+        - lib/
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - make
+        - cmake
+        - ant
+        - yarn
+        - wget
+        - ripgrep
+        - coreutils  # [osx]
+        - patchelf  # [linux]
+        - unzip
+        - nodejs =14
+        - {{ cdt('mesa-libgl-devel') }}  # [linux]
+        - {{ cdt('mesa-dri-drivers') }}  # [linux]
+        - {{ cdt('libselinux') }}  # [linux]
+        - {{ cdt('libxdamage') }}  # [linux]
+        - {{ cdt('libxxf86vm') }}  # [linux]
+        - {{ cdt('libxext') }}     # [linux]
+      host:
+        - r-base
+        - r-purrr
+        - r-rmarkdown
+        - r-renv
+        - r-testthat
+        - r-xml2
+        - r-yaml
+        - r-rstudioapi
+        - libxkbcommon  # [linux]
+        - xorg-libxcomposite  # [linux]
+        - xorg-libxcursor  # [linux]
+        - xorg-libxscrnsaver  # [linux]
+        - xorg-libxi  # [linux]
+        - xorg-libxtst  # [linux]
+        - openjdk
+        - boost-cpp
+        - soci-postgresql
+        - soci-core
+        - soci-sqlite
+        - libpq
+        - qt
+        - libuuid  # [linux]
+        - bzip2
+        - zlib
+        - pango
+        - sqlite
+        - libcups  # [linux]
+        - openssl
+        - hunspell
+        - hunspell-en
+        - pandoc
+        - mathjax <3.0
+        - yaml-cpp
+        - jpeg  # [osx]
+        - libpng  # [osx]
+      run:
+        - r-base
+        - r-purrr
+        - r-rmarkdown
+        - r-renv
+        - r-testthat
+        - r-xml2
+        - r-yaml
+        - r-rstudioapi
+        - libxkbcommon  # [linux]
+        - libxkbcommon  # [linux]
+        - xorg-libxcomposite  # [linux]
+        - xorg-libxcursor  # [linux]
+        - xorg-libxscrnsaver  # [linux]
+        - xorg-libxi  # [linux]
+        - xorg-libxtst  # [linux]
+        - openjdk
+        - boost-cpp
+        - soci-postgresql
+        - soci-core
+        - soci-sqlite
+        - libpq
+        - qt
+        - libuuid  # [linux]
+        - bzip2
+        - zlib
+        - pango
+        - sqlite
+        - libcups  # [linux]
+        - openssl
+        - hunspell
+        - hunspell-en
+        - pandoc
+        - mathjax <3.0
+        - yaml-cpp
+        - jpeg  # [osx]
+        - libpng  # [osx]
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/rstudio/bin/rstudio  # [linux]
+        - sh -c "xvfb-run -s '-screen 0 1200x800x24' ${PREFIX}/bin/rstudio --no-sandbox --run-diagnostics; cat ~/rstudio-diagnostics/diagnostics-report.txt"  # [linux]
+        - test -f ${PREFIX}/lib/rstudio/RStudio.app/Contents/MacOS/RStudio  # [osx]
+
+about:
+  home: www.rstudio.com/products/rstudio/
+  license: GPL-3.0-or-later
+  license_file: COPYING
+  doc_url: https://github.com/rstudio/rstudio
+  summary: RStudio is an integrated development environment (IDE) for R
```

Comment:
You have two abouts. Maybe leftovers from the multi-output version?

Reply:
I vote to keep the multi-output structure. Once we figure out what to do with pam we could/should also package rstudio-server in the same recipe, and keeping the multi-output from the get-go would avoid the indentation-churn.